expected-result-generator/bin/generate_results.py

```diff
@@ -10,8 +10,11 @@
 import argparse
 import subprocess
 import os
+import glob
 
-CLASSPATH = ".:../build/libs/expected-result-generator-all.jar"
+jar_file_list = glob.glob("../build/libs/expected-result-generator-all*.jar")
```

Comment:
You should not use * here.  You can easily end up with multiple versions of
jar in the build directory and then you will have messed up classpath.
Generally it would be best to have build target which builds distribution
package with both running script and proper jar file.

On Tue, Jun 23, 2015, 22:10 Brian Rickman <notifications@github.com> wrote:

> In expected-result-generator/bin/generate_results.py
> <https://github.com/prestodb/tempto/pull/12#discussion_r33085717>:
>
> >
> > -CLASSPATH = ".:../build/libs/expected-result-generator-all.jar"
> > +jar_file_list = glob.glob("../build/libs/expected-result-generator-all*.jar")
>
> Updated description to show that this change is intentional.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prestodb/tempto/pull/12/files#r33085717>.
>